client/app/models/collection.js

```diff
@@ -7,6 +7,7 @@ export default DS.Model.extend({
     created_by: DS.belongsTo('user'),
     dateCreated: DS.attr('date'),
     dateUpdated: DS.attr('date'),
+    settings: DS.attr('object'),
```

Comment:
In the `Collection` model, `settings` is a text field at the moment because SQLite does not have json fields. So this should be a string field that contains stringified JSON